main/modes/platformer/entity.c

```diff
@@ -568,9 +476,20 @@ void despawnWhenOffscreen(entity_t *self)
 {
     if (
         (self->x >> SUBPIXEL_RESOLUTION) < (self->tilemap->mapOffsetX - DESPAWN_THRESHOLD) ||
-        (self->x >> SUBPIXEL_RESOLUTION) > (self->tilemap->mapOffsetX + TILEMAP_DISPLAY_WIDTH_PIXELS + DESPAWN_THRESHOLD) ||
-        (self->y >> SUBPIXEL_RESOLUTION) < (self->tilemap->mapOffsetY - DESPAWN_THRESHOLD) ||
-        (self->y >> SUBPIXEL_RESOLUTION) > (self->tilemap->mapOffsetY + TILEMAP_DISPLAY_HEIGHT_PIXELS + DESPAWN_THRESHOLD))
+        (self->x >> SUBPIXEL_RESOLUTION) > (self->tilemap->mapOffsetX + TILEMAP_DISPLAY_WIDTH_PIXELS + DESPAWN_THRESHOLD)
+    )
+    {
+        destroyEntity(self, true);
+    }
+
+    if (self->y > 63616){
```

Comment:
This is an oddly specific number

Reply:
I need to #define this. 
max uint16_t - (screen height << subpixel resolution)